Eatery/Models/EaterySwipeData.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  EaterySwipeData.swift
+//  Eatery
+//
+//  Created by Ethan Fine on 5/5/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Foundation
+
+class EaterySwipeData {
+    
+    private var swipeDataByHours = [Int : [SwipeDataPoint]]()
+  
+    init(swipeDataPoints: [SwipeDataPoint]) {
+        mapSwipeDataPointToHours(swipeDataPoints: swipeDataPoints)
+    }
+    
+    private func mapSwipeDataPointToHours(swipeDataPoints: [SwipeDataPoint]) {
+        for swipeDataPoint in swipeDataPoints {
+            if swipeDataForHour(hour: swipeDataPoint.militaryHour) == nil {
+                swipeDataByHours[swipeDataPoint.militaryHour] = [swipeDataPoint]
+            } else {
+                swipeDataByHours[swipeDataPoint.militaryHour]!.append(swipeDataPoint)
+            }
+        }
+    }
+    
+    func swipeDataForHour(hour: Int) -> [SwipeDataPoint]? {
```

Comment:
If `swipeDataByHours` is immutable, we can remove this method and make `swipeDataByHours` internal.

Reply:
can we rename to `getSwipeDataForHour`